CONTRIBUTING.md

```diff
@@ -126,10 +126,17 @@ pytest keras_nlp/keras_nlp/integration_tests/import_test.py -k="import"
 
 ### Run all tests
 
-You can run the unit tests for KerasNLP by running:
+You can run the testing we run continuously for the repository with:
```

Comment:
What does this comment update refer to? Is this something github is handling for us?

Reply:
Yeah, I just meant you can run the same test we run "on the cloud" for each master commit/PR by just running `pytest`. And yes, github does run these for us, those are the "checks" that run for each PR, configured in the `.github` directory in the repo.

What's a clearer phrasing for this?

Reply:
Just dropped the phrasing about continuously.

Reply:
No worries, thanks for the clarification